### PR TITLE
examples: better practice for view types in todos

### PR DESCRIPTION
**Pull request by gbj**
See discussion in #933.

### CODE REVIEW

File: examples/todo_app_sqlite/src/todo.rs [hunk 1]
```diff
@@ -175,9 +175,9 @@ pub fn Todos(cx: Scope) -> impl IntoView {
                                                             </ActionForm>
                                                         </li>
                                                     }
-                                                    .into_any()
                                                 })
                                                 .collect::<Vec<_>>()
+                                                .into_view(cx)
```
Comment (ModProg) [inline examples/todo_app_sqlite/src/todo.rs:180]: Should it be possible to `.collect()` directly into the view here?

Comment (ModProg) [reply examples/todo_app_sqlite/src/todo.rs:180]: my bad, `into_view` needs a Scope, and `.collect()` cannot provide that.

Comment (gbj) [reply examples/todo_app_sqlite/src/todo.rs:180]: All good! Thanks for looking at it :-)

Comment (ModProg) [reply examples/todo_app_sqlite/src/todo.rs:180]: Scratch that, while `Fragment::into_view` takes a `Scope` it is not used in the whole progress. So it should be possible to implement `FromIterator<View> for View`.
A naive implementation would just copy the code from `Fragment::into_view()` and `ComponentRepr::into_view()`:

```rs
impl FromIterator<View> for View {
    fn from_iter<T: IntoIterator<Item = View>>(iter: T) -> Self {
        let fragment = iter.into_iter().collect::<Fragment>();
        let mut frag = ComponentRepr::new_with_id("", fragment.id().clone());

        #[cfg(debug_assertions)]
        {
            frag.view_marker = fragment.view_marker;
        }

        frag.children = fragment.nodes;
        #[cfg(all(target_arch = "wasm32", feature = "web"))]
        if !HydrationCtx::is_hydrating() {
            for child in &self.children {
                mount_child(MountKind::Before(&self.closing.node), child);
            }
        }

        View::Component(frag)
    }
}

```

**Pull request merged**